content/_data/changelogs/lts.yml

```diff
@@ -10262,7 +10262,7 @@
     message: Important security fixes.
     references:
       - url: /security/advisory/2024-01-24/
-        title: security advisory
+        title: 2024-01-24 security advisory
```

Comment:
```suggestion
        title: security advisory
```

The date is only useful when it's different from the release date, as will be the case for 2.440.1.